content/en/docs/reference/setup-tools/kubeadm/kubeadm-reset.md

```diff
@@ -34,6 +34,36 @@ etcdctl del "" --prefix
 
 See the [etcd documentation](https://github.com/coreos/etcd/tree/master/etcdctl) for more information.
 
+### CNI configuration cleanup
+
+CNI plugins use the directory `/etc/cni/net.d` to store their configuration.
+The `kubeadm reset` command does not cleanup that directory. Backup the directory if needed.
+Then to remove it, execute the following command:
+
+```bash
+sudo rm -rf /etc/cni/net.d
+```
+
+### Network traffic rules cleanup
+
+The `kubeadm reset` command does not clean any iptables, nftables or IPVS rules applied
+to the host by kube-proxy. If you wish to perform this cleanup, you can use the same
+kube-proxy container which was used in your cluster and the `--cleanup` flag of the
+kube-proxy binary:
+
```

Comment:
Well ... the above paragraph is clear wrt the "how" aspect of rules cleanup,
can we provide some hints on the "why" aspect?

Similar to the comment above, as a user, I want to know whether this operation is necessary, and what bad things will happen if I skip this step.
Manually running a container on each and every node is not fun, anyway.

Reply:
@danwinship can provide the full context as a kube-proxy owner, but from my experience there is no issue leaving the network rules on the node even if the node is later reused in another cluster. they facilitate the cluster and kube-proxy manages them.

> A control loop ensures that the rules on each node are reliably synchronized with the Service and EndpointSlice state as indicated by the API server.

https://kubernetes.io/docs/reference/networking/virtual-ips/
i could add a clarification note and link to the above page.